examples/config/default/go-tc-counter/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base/go-tc-counter
+
+# Patch the deployment.yaml to change container image in Daemonset
+# to new tag on the image.
+images:
+- name: quay.io/bpfd-userspace/go-tc-counter
+  newName: quay.io/bpfd-userspace/go-tc-counter
+  newTag: latest
```

Comment:
For consistency here should we default to the "main" tag?  (Now we automatically push these to :main in gh actions`

Reply:
That's just the default, and it means "main" branch! I'm happy to change everything to latest we'd just need to do it here -> https://github.com/bpfd-dev/bpfd/blob/main/bpfd-operator/Makefile#L50 too

Reply:
I don't care, but there are 52 matches across 14 files of go-*-counter:latest in my branch. Then there a several instances of `xdp_pass:latest`, `tc_pass:latest` and `tracepoint:latest` which we aren't updating, so now some are using `main` and some are using `latest`.

Reply:
Lol then lets just update all the core bpfd stuff to latest :)

Reply:
There should be a lot less using `:main`